apps/dsp-app/src/app/project/list/list-item/list-item.component.ts

```diff
@@ -1,165 +1,54 @@
-import {
-  ChangeDetectionStrategy,
-  ChangeDetectorRef,
-  Component,
-  EventEmitter,
-  Input,
-  OnInit,
-  Output,
-} from '@angular/core';
-import { ListNode, ListResponse, RepositionChildNodeRequest } from '@dasch-swiss/dsp-js';
+import { ChangeDetectorRef, Component, Input, OnInit } from '@angular/core';
+import { ListChildNodeResponse, ListNode, ListResponse, StringLiteral } from '@dasch-swiss/dsp-js';
 import { ListApiService } from '@dasch-swiss/vre/shared/app-api';
-import { take } from 'rxjs/operators';
-import { ListNodeOperation } from '../list-item-form/list-item-form.component';
+import { startWith, switchMap } from 'rxjs/operators';
+import { ListItemService } from './list-item.service';
 
 @Component({
-  changeDetection: ChangeDetectionStrategy.OnPush,
   selector: 'app-list-item',
-  templateUrl: './list-item.component.html',
-  styleUrls: ['./list-item.component.scss'],
+  template: `
+    <app-list-item-element
+      *ngFor="let child of children; let index = index; let first = first; let last = last"
```

Comment:
add trackBy function to avoid looping through all DOM elements during change detection